api/lib/searchOperators.js

```diff
@@ -0,0 +1,98 @@
+module.exports = {
+  angToQuery: PageNo => {
+    const hasOperatorsRegEx = /[<>+-]/gm;
+
+    if (!PageNo.match(hasOperatorsRegEx)) {
+      return {
+        q: 'v.PageNo = ?',
+        parameters: [PageNo],
+        least: PageNo,
+        most: PageNo,
+        totalPages: 1,
+      };
+    }
+
+    const qOR = [];
+    const qAND = [];
+    const qIn = [];
+    const parameters = [];
+    let least = 0;
+    let most = 0;
+
+    const ltgtRegEx = /[<>][0-9]+/gm;
+
+    const ltgt = PageNo.match(ltgtRegEx);
+    if (ltgt) {
+      let loopnum = 0;
+      ltgt.forEach(num => {
+        if (!num.match(/-$/)) {
+          loopnum = num.slice(1);
+          parameters.push(loopnum);
+          const operator = num.slice(0, 1);
+          qAND.push(`v.PageNo ${operator} ?`);
+          if (loopnum < least || loopnum === 0) {
```

Comment:
`loopnum` at this point is a string since you assigned it `num.slice(1)`. should you convert it back to a number when you are doing these comparisons? your second condition `loopnum === 0` will alway be false otherwise since `===` doesn't do type coercion.

Reply:
how do I do this?  parseInt?

Reply:
I'd do  `if (loopnum < least || loopnum === '0')`